docs/commands.md

```diff
@@ -15,6 +15,7 @@ witness run [cmd] [flags]
 ```
       --archivista-server string                      URL of the Archivista server to store or retrieve attestations (default "https://archivista.testifysec.io")
   -a, --attestations strings                          Attestations to record ('product' and 'material' are always recorded) (default [environment,git])
+      --attestor-link-export                          Export the link attestation to its own file
```

Comment:
Given that `--outfile` is slightly different naming style to `--attestor-link-export`, it would maybe be good to align these two flags for better UX.

Reply:
This is flag is automatically added from the attestor option registry. But, definitely want to think through the UX on this more.